.github/CONTRIBUTING.md

```diff
@@ -25,6 +26,12 @@ If you're not used to this workflow with git, you can start with some [docs from
 
 ## Tests
 
+You have the option to test your changes locally by running the pipeline. For receiving warnings about process selectors and other `debug` information, it is recommended to use the debug profile. Execute all the tests with the following command:
+
+```bash
+nf-test test --profile debug,test,docker --verbose
```

Comment:
Does this work?

Reply:
There is no nf-test implemented in the pipeline so far. And removing it from the file would lead to failing of `nf-core lint` due to changes in a file that should not be changed. 

BTW: Thanks for the review 🙂

Reply:
Are you sure the linter will fail? I removed corresponding lines in `bamtofastq` and the linter seems to be okay with that.

https://github.com/nf-core/bamtofastq/pull/85

Reply:
Yeah, the linter does check to see if this file has changed from template. But @jannikseidelQBiC you can always skip that check, by adding CONTRIBUTING.md to files_unchanged section in your .nf-core.yml like here https://github.com/nf-core/bamtofastq/blob/master/.nf-core.yml

Reply:
It would be good to get nf-test to work for the default test profile at least.